compatibility_lib/compatibility_lib/compatibility_store.py

```diff
@@ -255,28 +263,18 @@ def get_self_compatibilities(self,
         install_name_to_package = {p.install_name: p for p in packages}
         package_to_result = {p: [] for p in packages}
 
-        query_params = [
-            bigquery.ArrayQueryParameter('install_names', 'STRING',
-                                         [package.install_name
-                                          for package in packages]),
-        ]
-        job_config = bigquery.QueryJobConfig()
-        job_config.query_parameters = query_params
-
-        query = ('SELECT * '
-                 'FROM {} s1 '
-                 'WHERE s1.install_name IN UNNEST(@install_names) '
-                 '      AND timestamp = ( '
-                 '          SELECT MAX(timestamp) '
-                 '          FROM {} s2 '
-                 '          WHERE s1.install_name = s2.install_name '
-                 '            AND s1.py_version = s2.py_version)'.format(
-                    self._self_table_id, self._self_table_id))
-
-        query_job = self._client.query(query, job_config=job_config)
-
-        for row in query_job:
-            p = install_name_to_package[row.install_name]
+        query = 'SELECT * FROM {}'.format(
```

Comment:
Using '*' means that your code will break if we add another column. Maybe that is OK.

Reply:
Shouldn't you only select the given packages, not all of them? And maybe only with the latest timestamp?

Reply:
Does `_filter_older_versions` still need to exist?

Reply:
Done, selected the given packages. The _filter_older_versions will be removed in a separate PR.

Reply:
Removed `_filter_older_versions`